### DISCUSSION TITLE
Equirectangular images for creating a point cloud?

### DISCUSSION THREAD

**Discussion by rowild** [openMVG/openMVG#2273] [Q&A] [answered]
Can OpenMVG handle equirectangluar images? I did not find a tutorial on that topic – if it can, what do I have to do?

Currently, I use the demo version of Metashape, which can handle equirectangluar images. Is there any comparison available on the results, whether metashape or openMVG is better / worse or faster / slower?

**Comment by pmoulon** [comment, marked as answer]
We can handle for reconstruction (SFM) but export for MVS (points cloud to 3rd party tool) is still experimental. 

See here for small dataset and instruction https://github.com/openMVG/Image_datasets/tree/main/360_dataset

I need to update instructions to map our last version update.

Regarding Speed. We claim to share knowledge on SfM but for sure can't compete on speed with commercial software but accuracy is still very good

**Comment by rowild** [comment]
Thank you! – I do not understand the answer though (I am a newbie). I would like to create a point cloud from my insta360, export it to a ply file and use it in Blender.

So I wonder what "export for MVS" means. Does this mean the point cloud, once generated, is nowhere saved? Will it be lost once I close the program again?

**Reply by pmoulon** [reply]
No worries on noobiness. We all are at some point. SFM structure from motion: find pose of camera with a very sparse point cloud. MVS Multiple View Stereovision extract dense point cloud and mesh and texture from know camera poses (output of SFM). OpenMVG is a SFM software. OpenMVS is a MVS software than OpenMVG can export it's data to.
